ui/shared/status/StatusChatInputReplyArea.qml

```diff
@@ -51,7 +51,7 @@ Rectangle {
         font.pixelSize: 13
         font.weight: Font.Normal
         // Eliding only works for PlainText: https://bugreports.qt.io/browse/QTBUG-16567
-        textFormat: Text.PlainText
+        textFormat: Text.RichText
```

Comment:
Ugh, we need to make a decision here. Either we support showing the images with rich text or we support eliding.
I think the images is more important, but we need to make sure that long text doesn't look too ugly

Reply:
That's interesting but also a pretty gnarly problem. I overlooked the comment but I found another place where eliding is a problem. After you've sent the reply, that quoted message doesn't have any eliding but can display the emojis as images.